.prettierrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
can we share these dotfiles into a common repo?

Reply:
what do you mean?

Reply:
I mean some common repo (ie. io-typescript-template) to put these files